plotting/map.py

```diff
@@ -107,37 +111,26 @@ def __load_quiver(self) -> bool:
         )
 
     def load_data(self):
-        distance = GeodesicDistance()
-        height = (
-            distance.measure(
-                (self.bounds[0], self.centroid[1]), (self.bounds[2], self.centroid[1])
-            )
-            * 1000
-            * 1.25
-        )
-        width = (
-            distance.measure(
-                (self.centroid[0], self.bounds[1]), (self.centroid[0], self.bounds[3])
-            )
-            * 1000
-            * 1.25
-        )
+
+        width_scale = 1.25
+        height_scale = 1.25
 
         if self.projection == "EPSG:32661":  # north pole projection
             near_pole, covers_pole = self.pole_proximity(self.points[0])
             blat = min(self.bounds[0], self.bounds[2])
             blat = 5 * np.floor(blat / 5)
 
             if self.centroid[0] > 80 or near_pole or covers_pole:
-                self.basemap = basemap.load_map(
-                    "npstere",
-                    self.centroid,
-                    height,
-                    width,
-                    min(self.bounds[0], self.bounds[2]),
+                self.plot_projection = ccrs.Stereographic(
+                    central_latitude=self.centroid[0],
+                    central_longitude=self.centroid[1],
                 )
+                width_scale = 1.5
```

Comment:
This factor was added to account for projection distortion at the poles and ensure that we select a wide enough range to display the selected area.